doc/obs_integration.md

```diff
@@ -72,6 +75,108 @@ use `*.gem` files which are not supported by OBS services. It uses the
+osc meta prj -F - home:$OBS_USER:branches:systemsmanagement:Agama:Staging
+```
+
+The branched package is still linked to the original package. This might cause
+conflicts after the original package is updated. To avoid this problem you
+should detach the branched package from the original repository:
+
+``` shell
+osc detachbranch home:$OBS_USER:branches:systemsmanagement:Agama:Staging agama-web-ui
+```
+
+If you want to also build the Live ISO from your modified packaged then you need
+to branch (and detach) also the `agama-live` package:
+
+``` shell
+osc branch systemsmanagement:Agama:Staging agama-live
+osc detachbranch home:$OBS_USER:branches:systemsmanagement:Agama:Staging agama-live
+```
+
+*Please delete your branched OBS project once you do not need it anymore, it
+will save quite some OBS build power.*
+
+#### GitHub Fork
+
+Then you can fork the [Agama](https://github.com/openSUSE/agama) repository at
+GitHub.
+
+When creating a fork GitHub by default copies only the `master` branch. It does
+not copy the other branches nor tags. The code supports this option, just be
+prepared that the package versions might be different than in the original Agama
+OBS repository because the tags are used to get the version number. If tags are
+missing the version will be set to the commit Unix time stamp followed by a
+short commit hash.
+
+If you want to have similar versions as the original packages then create a full
+fork including all branches and tags (unselect the "Copy the master branch only"
+option).
+
+#### Configuring the GitHub Actions
+
+The GitHub Actions needs some configuration to allow the automatic submission.
+
+1. Go to the Settings -> Secrets and variables -> Actions -> New Repository
+   Secret option in your Agama GitHub fork. Create two secrets with names
+   `OBS_USER` and `OBS_PASSWORD` and set them to your OBS credentials.
+
+   Tip: If you do not want to put your credentials for your main OBS account
+   to GitHub then you might create a new separate testing OBS account.
+
+2. Switch to "Variables" tabs and click "New Repository Variable".
+   Create `OBS_PROJECT` variable with name of your OBS branch project
+   ("home:$OBS_USER:branches:systemsmanagement:Agama:Staging"). If the variable
+   is not created or is empty the autosubmission is disabled.
+
+3. Enable the GitHub Actions in the "Actions" tab
+
+#### Triggering the Rebuild
+
+*The autosubmission only works in the `master` branch in your fork, not in any
+other branch!*
+
+To trigger an update and rebuild of your package with the new sources just push
+a commit to the `master` branch.
+
+Alternatively you can trigger the package submission manually. Go to the
+"Action" tab in GitHub, select the respective "Submit" action in the left side
+bar and in the main area click the "Run workflow" selector. Then in the popup
+keep the default `master` branch and click the "Run workflow" button.
+
 ### Package Versioning
```

Comment:
Perhaps this section should be one level up. I know it is not part of this PR, but given that you are touching the file... :-)